src/hooks/useFilterProposals.js

```diff
@@ -18,11 +18,11 @@ import { checkInitialLetters } from '../lib/search-utils'
 const NULL_FILTER_STATE = -1
 const STAKE_STATUS_FILTER_SUPPORTED = 1
 const STAKE_STATUS_FILTER_NOT_SUPPORTED = 2
-const EXECUTION_STATUS_FILTER_OPEN = 0
-const EXECUTION_STATUS_FILTER_ACCEPTED = 1
-const EXECUTION_STATUS_FILTER_CANCELLED = 2
-const TYPE_FILTER_FUNDING = 0
-const TYPE_FILTER_SIGNALING = 1
+const EXECUTION_STATUS_FILTER_OPEN = 1
```

Comment:
maybe we can alphabetize those constants

Reply:
In this case i think its preferable the order to be by type of filter (given by the number) wdyt?